.github/workflows/ci.yml

```diff
@@ -91,13 +93,12 @@ jobs:
 
   test-all-action:
     name: "All - Github Action Test"
-    runs-on: "${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
         os:
           - "ubuntu-latest"
-          - "macos-latest"
```

Comment:
We need to have this being tested also on macos.

Reply:
Tailscale support isn't there
```
Error: Support Linux Only
Error: Process completed with exit code 1.
```